packages/vm/src/context.rs

```diff
@@ -18,7 +18,6 @@ use cosmwasm_std::{
     QuerierResponse, QueryRequest, Storage,
 };
 
-#[cfg(feature = "iterator")]
```

Comment:
why did you remove this?

Reply:
The imports under it are needed in your PR, but they are not available without this feature enabled. this caused a compilation fail when not using this feature.